contracts/0.8.9/WithdrawalQueue.sol

```diff
@@ -91,24 +93,19 @@ contract WithdrawalQueue {
     /// - N after upgrading contract from the previous version (after calling finalize_vN())
     bytes32 internal constant CONTRACT_VERSION_POSITION = keccak256("lido.WithdrawalQueue.contractVersion");
 
-    /// Lido DAO Agent contract address
-    /// Used to call administrative levers
-    bytes32 internal constant LIDO_DAO_AGENT_POSITION = keccak256("lido.WithdrawalQueue.lidoDAOAgent");
-
-    /// Requests placement resume/pause control storage slot
-    bytes32 internal constant REQUESTS_PLACEMENT_RESUMED_POSITION =
-        keccak256("lido.WithdrawalQueue.requestsPlacementResumed");
+    /// Withdrawal queue resume/pause control storage slot
+    bytes32 internal constant RESUMED_POSITION = keccak256("lido.WithdrawalQueue.resumed");
```

Comment:
Should claims be paused as well? They are not not. If not, I like full name with reference to what part of functionality if paused.

Reply:
Don't think that it worth it. At least in the name. REQUESTS_PLACEMENT_AND_FINALIZATION_RESUMED_POSITION looks too monstrous from my PoV. But I'd like to mention it in the docs. Thanks.